### ISSUE TITLE
RangeError: start offset of Float32Array should be a multiple of 4

### ISSUE THREAD

**Issue by looeee** [donmccurdy/glTF-Transform#256]
Attempting to load one of the Google Poly models gives the above error. 

Loading with this script:

``` js
import { NodeIO } from "@gltf-transform/core";

const io = new NodeIO()

const document = io.read("toy robot_5tz5/model.gltf");
```

The model does load and look correct in gltf-viewer, however the validation page reports some errors related to accessors not be multiples of 4 length. 

Here's the model:
[toy robot_5tzt.zip](https://github.com/donmccurdy/glTF-Transform/files/6520938/toy.robot_5tzt.zip)

**Comment by donmccurdy** [comment]
Hm, that's a fairly serious error in the model (potentially costs clients an extra copy operation over all the vertex data, besides being out of validation), but I guess this is coming from Google Blocks and the source can't really be fixed at this point?

Will see if there's some way to patch the loading process to handle this, and then the output from glTF-Transform should automatically have the right accessor padding.

**Comment by donmccurdy** [comment]
Fixed in https://github.com/donmccurdy/glTF-Transform/commit/486263679c2a29669db4543c286b53477c8fe803, to be released with v0.11.1. At some point in the future I might optimize things to avoid ArrayBuffer copies while loading, but it was already doing one on this data anyway, so no harm in handling incorrect FloatArray offsets for now. Might be worth a simple `gltf-transform cp a.glb b.glb` on these files to iron out the validation issues for archive.

**Comment by looeee** [comment]
> I guess this is coming from Google Blocks and the source can't really be fixed at this point?

Well, running it through gltfpack does fix it. 

However I had decided not to do that for the models as it adds another extension requirement (KHR_mesh_quantization) and I want to keep compatibility as high as possible. Also I have found some models that are broken by gltfpack so I'm wary of blindly running it all thousands of models - way to many to check manually.

**Comment by donmccurdy** [comment]
> Does this mean from v0.11.1 this error will be automatically fixed?

Yeah, just read/writing the file with no other changes should fix this issue once v0.11.1 is out.

**Comment by donmccurdy** [comment]
In general it's probably necessary to put models reporting this error through https://github.khronos.org/glTF-Validator/ – the validator should detect most major problems with the model. Ideally validation issues can be reported to the software creating the model and fixed there, but workarounds to avoid crashing on invalid inputs might be possible within glTF Transform, in limited cases.
